### PR TITLE
created a more comprehensive description about Installation, Usage, and Contribution Guidelines

### PR DESCRIPTION
**Pull request by Atharv-2004**
This pull request enhances the README.md file by adding detailed sections on the project overview, installation instructions, usage examples, and contribution guidelines. The updates include:

Project Overview: Provides a detailed explanation of Apertium’s purpose, features, and design philosophy.
Installation Instructions: Step-by-step guide on how to install Apertium, including prerequisites and verification steps.
Usage Examples: Examples of how to use Apertium for translating text and links to advanced documentation.
Contribution Guidelines: Information on how to contribute to both the language-agnostic tools and language data, along with a note on our current focus on rule-based systems.
These changes aim to make it easier for new users and contributors to understand and engage with the Apertium project.

### CODE REVIEW

File: README.md
Comment (mr-martian) [inline README.md:1]: Please edit `README` and leave `README.md` as a symlink.

**Review by mr-martian — CHANGES_REQUESTED** [review]

File: README.md
Comment (mr-martian) [reply README.md:1]: Also you've pasted the same text twice

Comment (Atharv-2004) [reply README.md:1]: @mr-martian can you please review it again?

File: README [hunk 1]
```diff
@@ -1,86 +1,117 @@
...
+    
+
+    You should see the help message for Apertium.
+
+## Usage
+
+To use Apertium for translating text, you can run the following command:
+
+```bash
+echo "Hello, world!" | apertium en-es# Apertium
```
Comment (mr-martian) [inline README:59]: you're still pasting the same text twice

you've also discarded what information was previously present about dependencies and binaries

**Review by mr-martian — CHANGES_REQUESTED** [review]

File: README (hunk 1 above)
Comment (Atharv-2004) [reply README]: hope this commit solves it all @mr-martian

**Comment by Atharv-2004** [comment]
@mr-martian can you please tell me what's still wrong?

**Pull request merged**